DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/README.md

```diff
@@ -63,13 +63,14 @@ You can also find more information about [troubleshooting build errors](/DirectP
 
 ### Performance
 
-Performance results are based on testing as of April 12, 2024 with fixed 55 [QR iterations](../pca/README.md#eigen-values-and-eigen-vectors-computation).
+Performance results are based on testing as of September 12, 2024 with fixed 55 [QR iterations](../pca/README.md#eigen-values-and-eigen-vectors-computation).
 
 > **Note**: Refer to the [Performance Disclaimers](/DirectProgramming/C++SYCL_FPGA/README.md#performance-disclaimers) section for important performance information.
 
-| Device                                            | Throughput
-|:---                                               |:---
-| Silicom FPGA SmartNIC N6011                       | 987 matrices/s for matrices of size 32 x 32
+| Device                      | Input size (Rows x Cols) | Throughput     |
+|-----------------------------|--------------------------|----------------|
+| Silicom FPGA SmartNIC N6011 | 32 x 32                  | 987 matrices/s |
+| Silicom FPGA SmartNIC N6011 | 128 x 32                 | 560 matrices/s |
```

Comment:
Can you also add the numbers you got using numpy, giving specs of the machine you ran the test on?

Reply:
I don't think we should report on the numpy numbers, since Intel legal requires high standards when reporting benchmark results on other peoples' tools.